docs/outlook/append-on-send.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Implement append on send in your Outlook add-in (preview)
+description: Learn how to implement the append-on-send feature in your Outlook add-in.
+ms.topic: article
+ms.date: 05/26/2020
+localization_priority: Normal
+---
+
+# Implement append on send in your Outlook add-in (preview)
+
+By the end of this walkthrough, you'll have an add-in that can insert a disclaimer when a message is sent.
```

Comment:
```suggestion
By the end of this walkthrough, you'll have an Outlook add-in that can insert a disclaimer when a message is sent.
```
FYI, Google crawls the first sentence of the first paragraph so always be sure to add SEO terms ("tags").